### DISCUSSION TITLE
publish huge vloume of maesage on the same subject

### DISCUSSION THREAD

**Discussion by qiaohe** [nats-io/nats-server#5932] [General]
when publishing huge volume of message on the same subject, The nats message has long latency (about 3minutes) any resones

**Comment by Jarema** [comment]
Is this core NATS? Or ther subject is part of JetStream stream?

I suspect the latter, as its when is ususlly happens when you publish to a stream without any backpressure control (by not handling max acks in flight or publishing to Stream using Core NATS publish, instead of JetStream publish)
